app/controllers/works_controller.rb

```diff
@@ -332,5 +332,17 @@ def process_updates
         render :edit, status: :unprocessable_entity
       end
     end
+
+    def params_collection_id
+      # Do not allow a nil for the collection id
+      @params_collection_id ||= begin
+        collection_id = params[:collection_id]
+        if collection_id.blank?
+          collection_id = current_user.default_collection.id
+          Honeybadger.notify("We got a nil collection as part of the parameters #{params} #{request}")
```

Comment:
Am I understanding this correctly that it's an error state that we want to be notified about, but that it also shouldn't happen?

Reply:
Yes, that is correct.  I am unsure how it occurred, so I want to log if it ever happens again...